force-app/main/default/classes/ServiceService.cls

```diff
@@ -11,19 +11,60 @@ public with sharing class ServiceService {
     public ServiceService() {
     }
 
+    @TestVisible
+    private FieldBucketSelector bucketSelector = new FieldBucketSelector();
+
     @TestVisible
     private FieldSetService fieldSetService = new FieldSetService();
 
     @TestVisible
     private ServiceSelector serviceSelector = new ServiceSelector();
 
+    @TestVisible
+    private ProgramEngagementService engagementService = new ProgramEngagementService();
```

Comment:
Not suggesting we change it but curious if it would be better to only include the service vs the service and the selector since the service calls the selector as well.